trivy-fs-scan/action.yml

```diff
@@ -0,0 +1,85 @@
+name: Trivy FS Scan
```

Comment:
wondering if we can re-use existing `trivy-image-scan` action for file system as well.

Reply:
Sure i'll update the existing action